server/chat/backend/agent/tools/cloud_tools.py

```
@@ -2373,36 +2379,73 @@ def _pinned_trigger(action_id: str = "", _pid=pinned_id, _fn=final_func, **kw):
+                        "edits (list of anchored search-and-replace edits: each has old_string + new_string; "
+                        "old_string must match the current file exactly, with enough surrounding context to be unique; "
+                        "set replace_all=true if you want every occurrence replaced; "
+                        "to CREATE a new file, use old_string='' with the full content in new_string), "
+                        "fix_description (what this fix does), root_cause_summary (why this change is needed). "
+                        "Optional: repo (workspace/repo_slug format), commit_message, branch."
+                    ),
+                    args_schema=BitbucketFixArgs,
+                ))
+            logging.info(
+                f"Added {len(_bb_tools) + (1 if _is_rca_background else 0)} Bitbucket tools for user "
+                f"{user_id} (rca_background={_is_rca_background}, pr_review={is_pr_review})"
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)